app/models/services/service_binding_operation.rb

```diff
@@ -1,7 +1,7 @@
 module VCAP::CloudController
   class ServiceBindingOperation < Sequel::Model
     export_attributes :state, :description, :type, :updated_at, :created_at
-
+    many_to_one :service_binding
```

Comment:
What's the effect of this change?

Reply:
allows us to use Factories to create Operations in tests like this: 
https://github.com/cloudfoundry/cloud_controller_ng/pull/3058/files#diff-f3725445b6d7d167e4273ed2a61ba2b541da978b8ea8f6e3b31a43f7e956ae9eR875

Reply:
Ah, thanks for the explanation.